### PR TITLE
Sender CSV content revisions

### PR DESCRIPTION
**Pull request by drew-usds**
This PR ...

Continues work on content revisions for CSV senders from PRs #3729 and #3732. 

Includes final updates after reviews with product, design, and O&O.

Test Steps:
1. *Include steps to test these changes*

## Changes
- Content and styling tweaks to content in `/getting-started/testing-facilities`

### Testing
- [x] Tested locally?
- [ ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [x] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [ ] Added tests?

### Process
- [ ] Are there licensing issues with any new dependencies introduced?
- [ ] Includes a summary of what a code reviewer should test/verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Fixes
- #3666

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?

### CODE REVIEW

File: frontend-react/src/pages/getting-started/testing-facilities/AccountRegistrationGuide.tsx [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+                    </li>
+                    <li className="usa-process-list__item">
+                        <h4 className="usa-process-list__heading">
+                            Accept the terms of service
+                        </h4>
+                        <ul className="margin-top-2">
+                            <li>
+                                A ReportStream representative will process your
+                                registration request and respond via email with
+                                a link to an online form.{" "}
```
Comment (TomNUSDS) [inline frontend-react/src/pages/getting-started/testing-facilities/AccountRegistrationGuide.tsx:113]: Did you mean to have `{" "}` for every item in this list? Seems better to have some style markup than depend on React to inject it.

**Review by TomNUSDS — APPROVED** [review]
Looks good to me. 

Stylistically, I don't care for the use of `{" "}` to add whitespace, but it seems to be a common practice (aka SimpleReport does it a lot as well).

File: frontend-react/src/pages/getting-started/testing-facilities/AccountRegistrationGuide.tsx (hunk 1 above)
Comment (drew-usds) [reply frontend-react/src/pages/getting-started/testing-facilities/AccountRegistrationGuide.tsx:137]: @TomNUSDS looks like the linter is adding that, I didn't any `{" "}` to the markup. Is that something we should change with the linter?

Comment (TomNUSDS) [reply frontend-react/src/pages/getting-started/testing-facilities/AccountRegistrationGuide.tsx:137]: Oh, interesting. I think the linter might be trying to preserve multiple spaces in the final output (which seems silly for html)? @sean-usds any thoughts?

Comment (drew-usds) [reply frontend-react/src/pages/getting-started/testing-facilities/AccountRegistrationGuide.tsx:137]: Actually I _kind of_ get why it's doing it, the linter is adding line breaks around things like `<a>` while also removing empty spaces around the text. It needs those `{" "}` to make the flow around other text work.

Without it you'd have:

```
Lorem ipsum
  <a href="/foo">
    LINK-TEXT
  </a>
dolor sit amet
```

That would translate to:

> Lorem ipsumLINK-TEXTdolor sit amet

**Review by sean-usds — APPROVED** [review]
🚀 🚀 🚀

**Pull request merged**